SingleMeshBlend/SingleMeshBlend/SingleBlendMeshDeformer.cpp

```diff
@@ -123,6 +123,7 @@ MStatus SingleBlendMeshDeformer::deform(MDataBlock & block, MItGeometry & iterat
 		MObject blendMeshValue{ block.inputValue(blendMesh).asMesh() };
 		MFnMesh blendMeshFn{ blendMeshValue };
 
+		CHECK_MSTATUS_AND_RETURN_IT(iterator.allPositions(taskData.vertexPositions));
```

Comment:
sorry I meant to cache only the deltas, not the input point, those you need to read every frame, you don't know what deformer you might have before yours.